ftw/labels/locales/update.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
We already provide a script to update the translation files, please use `bin/i18n-build` for this.